test/setup.js

```diff
@@ -100,5 +92,12 @@ test._.mixin({
 	}
 }, { chain: false });
 
-// Exports
-module.exports = test;
+// See https://github.com/shouldjs/should.js/issues/41
+Object.defineProperty(global, 'should', { value: chai.should() });
+global.chai = chai;
+global.expect = chai.expect;
+global.sinon = sinon;
+global.sinonChai = sinonChai;
+global.sandbox = sinon.sandbox.create();
```

Comment:
This doesn't match the documented way of creating sandboxes: http://sinonjs.org/releases/v3.2.1/sandbox/